cli/src/tunnels/service_windows.rs

```diff
@@ -241,6 +253,13 @@ fn service_main(_arguments: Vec<OsString>) -> Result<(), AnyError> {
 		})
 		.map_err(|e| wrap(e, "error marking service as running"))?;
 
+	info!(service.log, "Starting service loop...");
+
+	let panic_log = service.log.clone();
+	std::panic::set_hook(Box::new(move |p| {
```

Comment:
Found this nice functionality when debugging. Windows services don't and can't capture stdout/err, so this lets us put panic information to the Logger (and therefore to the log file).